packages/core/types/src/modules/entity-validator.ts

```diff
@@ -4,12 +4,10 @@ import type * as UID from '../uid';
 import type * as EntityService from './entity-service';
 
 type Entity = {
-  id: ID;
+  id: number;
```

Comment:
Are we sure this can't ever be string?

Reply:
Yes I think so. The entity validator works with entities not documents everywhere